src/tests/build.sh

```diff
@@ -170,7 +170,7 @@ precompile_coreroot_fx()
         __NumProc=$(nproc --all)
     fi
 
-    local crossgenCmd="\"$__DotNetCli\" \"$CORE_ROOT/R2RTest/R2RTest.dll\" compile-framework -cr \"$CORE_ROOT\" --output-directory \"$CORE_ROOT/crossgen.out\" --target-arch $__BuildArch -dop $__NumProc"
+    local crossgenCmd="\"$__DotNetCli\" \"$CORE_ROOT/R2RTest/R2RTest.dll\" compile-framework -cr \"$CORE_ROOT\" --output-directory \"$CORE_ROOT/crossgen.out\" --large-bubble --release --target-arch $__BuildArch -dop $__NumProc"
```

Comment:
I think the `--nocleanup` should be added here too, right?

Reply:
Sure, thanks for reminding me!